gns3/ui/general_preferences_page_ui.py

```diff
@@ -25,6 +24,7 @@ def setupUi(self, GeneralPreferencesPageWidget):
         self.uiGeneralTab = QtWidgets.QWidget()
         self.uiGeneralTab.setObjectName("uiGeneralTab")
         self.verticalLayout_4 = QtWidgets.QVBoxLayout(self.uiGeneralTab)
+        self.verticalLayout_4.setContentsMargins(0, 0, 0, 0)
```

Comment:
Why your pyuic set margins to 0?

No margins:

![no_margins](https://cloud.githubusercontent.com/assets/4752614/16254100/e6811d20-37f9-11e6-92c2-e4da0367f181.png)

Margins:

![margins](https://cloud.githubusercontent.com/assets/4752614/16254113/f6f12d12-37f9-11e6-84a8-581aa78de277.png)